explainer.md

```diff
@@ -74,15 +75,20 @@ navigator.vr.getDevices().then(devices => {
   // permissions by a parent frame.
 });
 ```
-### Detecting and advertising VR mode
 
-If a `VRDevice` is available and has the appropriate capabilities the page will usually want to add some UI to trigger activation of "VR Presentation Mode", where the page can begin sending imagery to the device. Testing to see if the device supports the capabilities the page needs is done via the `supportsSession` call, which takes a dictionary of the desired functionality and returns whether or not the device can create a session supporting them. Querying for support this way is necessary because it allows the application to detect what VR features are available without actually engaging the sensors or beginning presentation, which can incur significant power or performance overhead on some systems and may have side effects such as launching a VR status tray or storefront.
+### Sessions
+
+A `VRDevice` indicates the presence of a VR hardware device but provides very little information about it outside of a name that could be used to select it from a list. In order to do anything that involves the hardware's presentation or tracking capabilities the page will need to request a [`VRSession`](https://w3c.github.io/webvr/#interface-vrsession) from the `VRDevice`.
```

Comment:
It's nice to somehow indicate that you are referencing a definition and link to that definition. This may not be easy to do in MD, but in the spec, you'd probably want to make "VR hardware device" a link.